README.md

```diff
@@ -37,25 +37,25 @@
 
 ## Nakadi Event Broker
 
-The goal of the Nakadi project (ნაკადი means "stream" in Georgian) is 
-to build an event broker infrastructure to:
+The goal of Nakadi (ნაკადი means "stream" in Georgian) is to provide an efficient event broker infrastructure that allows convenient development of event-driven applications and asynchronous microservices. 
 
-*  Allow event exchange via a RESTful [API](/api/nakadi-event-bus-api.yaml)
-*  Enable convenient development of event-driven applications
-*  Securely and efficiently publish and consume events
+- **HTTP API with OAuth**. Publish and consume events via a [RESTful API](/api/nakadi-event-bus-api.yaml) making integration with any technology stack easy and allowing microservices teams to maintain service boundaries. Access to the API can be managed and secured using OAuth scopes.
 
-Some additional requirements for the project are to provide:
+- **Standard Event Types**. Event types for business processes and data changes can be defined based on common primitives for identity, timestamps, event types, and causality. 
```

Comment:
Strictly speaking, the event type formats are not really part of Nakadi. We are including it in the API, because there is not other place to discuss and legalise these formats.

Maybe one can mention, that the standard event types are really an optional feature of Nakadi